playbooks/luna-devel.yml

```diff
@@ -2,9 +2,6 @@
 - import_playbook: katello_devel.yml
   vars:
     foreman_installer_options:
-      - "--katello-devel-github-username '{{ katello_devel_github_username }}'"
-      - "--katello-devel-enable-ostree=true"
```

Comment:
https://github.com/theforeman/forklift/pull/1311 is open to really remove it. I'd prefer to get that finished.